### PR TITLE
docs(components): bring back Card header with action example

### PR DESCRIPTION
**Pull request by chris-at-jobber**
## Motivations

The Card component now has header props that allow for an action. In a commit in the PR to introduce the feature, the documentation of this feature appears to have been [accidentally deleted](https://github.com/GetJobber/atlantis/pull/1026/commits/4e2d677d806f041ded0efcaf4b8f3b07cd412d70).

### Added

- example and guidance for header actions

## Testing

View Card docs and see there is a section for the header action example.

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/components/src/Card/Card.mdx [hunk 1]
```diff
@@ -70,6 +70,42 @@ Card titles should be sentence-cased.
...
+      title: "Banana varieties",
+      action: {
+        label: "New Banana",
+        icon: "add",
+        onClick: () => {
+          alert("New banana added");
+        },
+      },
+    }}
+  >
```
[the remark this answers is not in the document]
Comment (chris-at-jobber) [reply packages/components/src/Card/Card.mdx:91]: image

[the remark this answers is not in the document]
Comment (chris-at-jobber) [reply packages/components/src/Card/Card.mdx:91]: ^ Note that the `Card` is not syntax-highlighted. When I delete the `onClick` the syntax highlighting comes back, but still no header 🤔 
image

File: packages/components/src/Card/Card.stories.mdx [hunk 2]
```diff
@@ -68,6 +92,105 @@ Card titles should be sentence-cased.
...
+    />
+  </Card>
+</Canvas>
+
+### Header with multiple actions
+
+Pass a Menu in as the header action to present the user with multiple actions
+related to the Card contents.
+
+<!--
```
Comment (chris-at-jobber) [inline packages/components/src/Card/Card.stories.mdx:140]: This was breaking the docs

File: packages/components/src/Card/Card.stories.mdx [hunk 3]
```diff
@@ -68,6 +92,105 @@ Card titles should be sentence-cased.
...
+### Header with action
+
+Pass a Button in as the header action to present the user with an action related
+to the Card contents.
+
+<Canvas>
+  <Card
+    header={{
+      title: "Services",
+      action: (
```
Comment (chris-at-jobber) [inline packages/components/src/Card/Card.stories.mdx:104]: I keep trying to pass this component in without the parentheses but it auto-adds them on save? :help:

**Comment by chris-at-jobber** [comment]
The Story now has the desired action, but I can't get the "non-Story" Canvas versions to render actions.
image

**Review by darryltec — APPROVED** [review]

**Pull request merged**